src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -88,15 +93,16 @@ public CursorsController(final CursorsService cursorsService,
         if (cursors.isEmpty()) {
             throw new CursorsAreEmptyException();
         }
-
+        final Scope commitScope = TracingService.activateSpan((Span) request.getAttribute("span"), false);
```

Comment:
can we have `TracingService.activateSpan` take `request`, and do all the casting within the method ?